src/main/java/org/apache/commons/csv/CSVRecord.java

```diff
@@ -144,6 +159,15 @@ public long getCharacterPosition() {
         return characterPosition;
     }
 
+    /**
+     * Returns the start byte of this record as a character byte in the source stream.
```

Comment:
Please follow the same Javadoc patterns as in other getter methods: A getter "Gets...".

Reply:
ping